docs/guides/jsdom.md

```diff
@@ -11,22 +11,52 @@ gets run *before* React's code is run.
 
 As a result, a standalone script like the one below is generally a good approach:
 
+`jsdom v10~`:
+
 ```js
 /* setup.js */
 
-var jsdom = require('jsdom').jsdom;
+const { JSDOM } = require('jsdom');
+const jsdom = new JSDOM('<!doctype html><html><body></body></html>');
+const { window } = jsdom;
+
+function copyProps(src, target) {
+  const props = Object.getOwnPropertyNames(src)
+    .filter(prop => typeof target[prop] === 'undefined')
```

Comment:
no, actually I meant to filter out ones *which are undefined at the `target` object*.

this is equivalent with below code (would you prefer this style?):

```js
  Object.getOwnPropertyNames(src).forEach((prop) => {
    if (typeof target[prop] === 'undefined') { // if not defined yet as the 'target's props
      Object.getOwnPropertyDescriptor(src, prop);
    }
  });
```